scripts/populate_preprint_providers.py

```diff
@@ -60,7 +59,13 @@ def set_or_update_m2m_fields(provider, emails, licenses, links, social_accounts)
         provider.licenses_acceptable.add(*licenses)
 
     for link in links:
-        obj, created = PreprintProviderLink.objects.get_or_create(**link)
+        obj, created = provider.links.update_or_create(
```

Comment:
Hmmm, you have a point there. 

Originally, `ExternalLink` was unique on url and description, which would avoid this problem entirely, but in making the model more generic I removed that since we don't enforce any sort of uniqueness on user profile websites (which will use `ExternalLink`). 

And in this case since `update_or_create` is called on the links for a given preprint provider, one provider won't  be able to update another's external links.